notebooks/algorithm-design/applications.ipynb

```diff
@@ -0,0 +1,1534 @@
+    "Here the eigenstates corresponding to $\\lambda_1 = \\lambda_2 = 4$ are not the same as the expected ones: $|\\phi_1\\rangle = \\frac{1}{\\sqrt{2}}(|00\\rangle - |11\\rangle)$ and $|\\phi_2\\rangle = \\frac{1}{\\sqrt{2}}(|01\\rangle + |10\\rangle)$. This happens precisely because as $\\lambda_1=\\lambda_2$, any complex linear combination of those $\\alpha_1 |\\phi_1\\rangle + \\alpha_2 |\\phi_2\\rangle = \\frac{1}{\\sqrt{2}}(\\alpha_1 |00\\rangle + \\alpha_2 |01\\rangle - \\alpha_2 |10\\rangle - \\alpha_1 |11\\rangle)\\equiv \\frac{1}{\\sqrt{2}}[\\alpha_1, \\alpha_2, -\\alpha_2, -\\alpha_1]$ is also a eigenstate with the same eigenvalue. That is exactly what we are seeing with these results."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Change betas"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As mentioned in the instances chapter, the values of $\\vec{\\beta}$ should be bigger than the difference between eigenvalues. Let us see what happens when they do not verify that condition."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vqd.betas = np.ones(4)\n",
+    "result = vqd.compute_eigenvalues(observable_2)\n",
+    "\n",
+    "## PRINT\n",
+    "optimal_circuits = [\n",
+    "    circuit.bind_parameters(parameters)\n",
+    "    for circuit, parameters in zip(result.optimal_circuits, result.optimal_parameters)\n",
+    "]\n",
+    "eigenstates = [Statevector.from_instruction(c) for c in optimal_circuits]\n",
+    "\n",
+    "for i, (eigenvalue, eigenstate) in enumerate(zip(result.eigenvalues, eigenstates)):\n",
+    "    eigenvalue = eigenvalue.real\n",
+    "    eigenstate = np.round(eigenstate.data, 3).tolist()\n",
+    "    print(f\"RESULT {i}:\")\n",
+    "    print(f\"  - {eigenvalue = :.3f}\")\n",
+    "    print(f\"  - {eigenstate = }\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This time, the optimizer returns the same state $|\\phi_0\\rangle = \\frac{1}{\\sqrt{2}}(|00\\rangle + |11\\rangle)$ as proposed solution to all eigenstates: which is clearly wrong. This happens because the betas were too small to penalize the minimum eigenstate in the successive cost functions; therefore, it was not excluded from the effective search space in later iterations of the algorithm, and always chosen as the best possible solution."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Variational algorithms with Qiskit Runtime"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## I'm following this how-to guide https://qiskit.org/documentation/partners/qiskit_ibm_runtime/how_to/noisy_simulators.html\n",
```

Comment:
Cells from here onwards will fail during CI as we can't run them automatically. You can add a `uses-hardware` tag to tell our scripts to skip them.